src/components/MyrTour.js

```diff
@@ -21,6 +21,7 @@ class MyrTour extends Component {
           steps={steps}
           maskClassName="mask"
           isOpen={this.state.isTourOpen}
+          onAfterOpen={this.props.isEditorClosed ? this.props.toggleEditor : ()=>{}}
```

Comment:
It looks like there is onAfterClose as well. Can you add that and have it toggle the editor closed after it opens it? It would also be good to not in the tour content how the user can do that themselves.